data/korath/korath.txt

```diff
@@ -254,24 +261,63 @@ fleet "Korath Home"
 	cargo 3
 	personality
 		timid opportunistic
-	variant 7
-		"Korath World-Ship"
-	variant 22
-		"Korath World-Ship B"
-	variant 7
-		"Korath World-Ship C"
 	variant 3
-		"Rano'erek"
-		"'olofez" 2
-		"'olofez (Expeller)" 2
+		"Lor'kas Ik 577"
+		"'olofez" 3
 	variant 1
+		"Kas'lor Ik 582"
+		"'olofez" 3
+	variant 1
+		"Lor'nag Ik 590 (Grab)"
+		"'olofez" 3
+
+fleet "Korath Defense"
+	government "Korath"
+	names "korath"
+	fighters "korath"
+	cargo 1
+	personality
+		disables plunders opportunistic harvests
+	variant 4
+		"Ra'gru Ik 618"
+	variant 4
+		"Ra'at Ik 621"
+		"'olofez" 9
+	variant 2
 		"Rano'erek (Shunt)"
 		"'olofez" 2
 		"'olofez (Expeller)" 2
 	variant 1
 		"Rano'erek (Heaver)"
 		"'olofez" 4
+	variant 3
+		"Palavret" 3
+		"'olofez" 6
+	variant 2
+		"Palavret" 2
+		"Palavret (Blaze)" 2
+		"'olofez" 8
+	variant 2
+		"Palavret (Ember Shunt)" 3
+		"'olofez" 6
+	variant 1
+		"Rano'erek"
+		"Palavret (Blaze)"
+		"'olofez" 4
+		"'olofez (Expeller)" 2
+	variant 2
+		"Ra'gru Ik 618 (Grab)"
+	variant 1
+		"Ra'gru Ik 618 (Storm)"
+	variant 1
+		"Ra'gru Ik 618"
+		"Palavret" 3
+		"'olofez" 6
 	variant 1
+		"Ra'at Ik 621 (Storm)"
+		"Palavret" 3
+		"'olofez" 11
+		"'olofez (Expeller)" 4
```

Comment:
Is this variant supposed to include the ships listed immediately below this?

Reply:
Looks like a line got eaten yeah
```suggestion
		"'olofez (Expeller)" 4
	variant 1
```